pkg/config/config.go

```diff
@@ -96,7 +96,7 @@ var (
 	// kyvernoConfigMapName is the Kyverno configmap name
 	kyvernoConfigMapName = osutils.GetEnvWithFallback("INIT_CONFIG", "kyverno")
 	// defaultExcludedUsernames are the usernames excluded by default when matching an incoming admission request
-	defaultExcludedUsernames []string = []string{"system:kube-scheduler"}
+	defaultExcludedUsernames []string
```

Comment:
Why overwrites it?

Reply:
How can this be overwritten? I see there is an option to `excludeUsername` but it is not present in the cm with `system:kube-schedule`.
```bash
$ k get cm -n kyverno kyverno -o yaml     
```
```yaml         
apiVersion: v1
data:
  defaultRegistry: docker.io
  enableDefaultRegistryMutation: "true"
  excludeBackgroundUsernames: system:serviceaccount:kyverno:kyverno-background-controller
  generateSuccessEvents: "false"
  resourceFilters: '[*,kyverno,*][Event,*,*][*,kube-system,*][*,kube-public,*][*,kube-node-lease,*][Node,*,*][APIService,*,*][TokenReview,*,*][SubjectAccessReview,*,*][SelfSubjectAccessReview,*,*][Binding,*,*][ReplicaSet,*,*][AdmissionReport,*,*][ClusterAdmissionReport,*,*][BackgroundScanReport,*,*][ClusterBackgroundScanReport,*,*][ClusterRole,*,kyverno:*][ClusterRoleBinding,*,kyverno:*][ServiceAccount,kyverno,kyverno-admission-controller][ConfigMap,kyverno,kyverno][ConfigMap,kyverno,kyverno-metrics][Deployment,kyverno,kyverno][Job,kyverno,kyverno-hook-pre-delete][NetworkPolicy,kyverno,kyverno][PodDisruptionBudget,kyverno,kyverno][Role,kyverno,kyverno:*][RoleBinding,kyverno,kyverno:*][Secret,kyverno,kyverno-svc.kyverno.svc.*][Service,kyverno,kyverno-svc][Service,kyverno,kyverno-svc-metrics][ServiceMonitor,kyverno,kyverno-svc-service-monitor][Pod,kyverno,kyverno-*]'
  webhooks: '[{"namespaceSelector": {"matchExpressions": [{"key":"kubernetes.io/metadata.name","operator":"NotIn","values":["kyverno"]}]}}]'
kind: ConfigMap
metadata:
  annotations:
    meta.helm.sh/release-name: kyverno
    meta.helm.sh/release-namespace: kyverno
  creationTimestamp: "2023-03-18T18:33:24Z"
  labels:
    app.kubernetes.io/component: config
    app.kubernetes.io/instance: kyverno
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/part-of: kyverno
    app.kubernetes.io/version: v3.0.0
    helm.sh/chart: kyverno-v3.0.0
  name: kyverno
  namespace: kyverno
  resourceVersion: "1078"
  uid: b9906f5f-ffb0-4b39-b667-588037a99683
```

Reply:
This is used to initialize the default cm configurations, setting it to empty does not exclude any usernames by default. No?

Reply:
@Vyom-Yadav ?

Reply:
> This is used to initialize the default cm configurations, setting it to empty does not exclude any usernames by default. No?

Yes. if it is empty, no username will be excluded by default.

Reply:
Why? We want this to be excluded by default `"system:kube-scheduler"`.

Reply:
> Why?

That's just the way code works.

> We want this to be excluded by default "system:kube-scheduler"

Binding is created by the scheduler, which is why it is needed to be removed from default. If you want I can add it to CM by default (which is modifiable).

Reply:
I see. Not excluding it might add extra work to Kyverno.

@eddycharly - wdyt?

Reply:
If a user wants to make the policy to work, they need to update two fields, `resourceFilters` and the `excludeusernames`(if the scheduler is excluded by default). 

Let's try with no exclude usernames to ease the UX, we can always revert it in case of any issue.

Reply:
As of now if the user wants to make it work, they have to remove `[Binding,*,*]` from the cm, rest nothing is required. Request from user `system:kube-scheduler` is processed and not ignored, we can change this if any problem occurs in the future.